apps/src/applab/applab.js

```diff
@@ -1445,7 +1413,7 @@ Applab.onPuzzleComplete = function (submit) {
         level: level.id,
         result: levelComplete,
         testResult: Applab.testResults,
-        submitted: !!submit,
+        submitted: submit ? "true" : false,
```

Comment:
The warning is mostly meant to document current behavior/alert us when behavior changes.

What you could do is change it so that if level.app === 'applab' || level.app === 'gamelab' then validateType('submitted', value, 'boolean') else currentLogic.

Then we're one step closer to just saying submitted is boolean across the board.